src/workerd/api/pyodide/pyodide.c++

```diff
@@ -27,6 +27,19 @@ void PyodideBundleManager::setPyodideBundleData(
       kj::mv(version), {.messageReader = kj::mv(messageReader), .bundle = bundle});
 }
 
+const kj::Maybe<kj::ArrayPtr<const unsigned char>> PyodidePackageManager::getPyodidePackage(
+    kj::StringPtr id) const {
+  KJ_IF_SOME(t, packages.lockShared()->find(id)) {
```

Comment:
Nit: This function can be simplified with just returning and calling kj::maybe's .map() to convert it into asPtr().